### DISCUSSION TITLE
How to connect if no F1 and F2 are available. wiring diagram necessary

### DISCUSSION THREAD

**Discussion by mikburnz** [omerfaruk-aran/esphome_samsung_hvac_bus#183] [General]
Can someone please post a picture of the wiring diagram where F1 and F2 are available? This is usually found under the cover. In my system, I only have 3 connections. Number 2 is marked as the communication port. I would like to compare it and find a solution.

![Klima1](https://github.com/user-attachments/assets/2346b132-0b6e-435e-94d9-7deeb9bd3090)
![Klima2](https://github.com/user-attachments/assets/52e852e6-ca7b-4235-968c-78fb9b897803)

**Comment by lanwin** [comment]
Hi, please look at the discussions. This topic is already discussed in another thread. Sadly this configuration did not work with this project. We need the F1/F2 bus that your device did not have.

**Reply by mikburnz** [reply]
I've seen other discussions. I'm quite sure its possible to connect. As you can see, Cable blk (2) is the com cable. I've seen pictures with F1 and F2 where F2 is the yellow green connectect. Thats why i need the wiring diagram of a functional setup. 

![image](https://github.com/user-attachments/assets/489acb6f-909f-4e5b-a1da-9188a053eade)

**Reply by lanwin** [reply]
Maybe. You can check what voltage you get there. But be careful. 1 and N should be 220/110v. If its below 12v, it can be an indication that you have an RS485. Then you can buy an cheap USB to RS485 adapter and check if you get any data dumped via serial output. If yes, you can try to connect the ESP here.
